cli/doRun_test.go

```diff
@@ -7,9 +7,23 @@ import (
 
 	"github.com/bitrise-io/bitrise/bitrise"
 	models "github.com/bitrise-io/bitrise/models/models_1_0_0"
+	envmanModels "github.com/bitrise-io/envman/models"
 )
 
 func Test0Steps1Workflows(t *testing.T) {
+	// Envman setup
+	if err := os.Setenv(bitrise.EnvstorePathEnvKey, bitrise.OutputEnvstorePath); err != nil {
```

Comment:
ever `Setenv` in the tests should have a `defer` block which `Unset`s the environment!